python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -498,8 +523,14 @@ async def poll_loop(self):
                     self.log.critical(f"Unhandled state: {self.summary_state}")
 
                 await asyncio.sleep(1)
+
+            except ModbusError as er:
+                await self.log_modbus_error(er, "While reconnecting:")
+                await self.disconnect()
+                await asyncio.sleep(5)
```

Comment:
Instead of using `sleep(5)` consider giving this time a representative name and make it either a module level constant or a class attribute.